Exercises.playground/Pages/Exercise2.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,24 @@
+//: [Previous](@previous)
+
+import Foundation
+
+func count(numbers: [Int], number: Int){
+    
+var greater=0
+var lower=0
+    
+for index in numbers{
```

Comment:
please use a better name instead of index for this variable , as you are using to get the value of the integer from the array, not its index